src/elements/play-pen/play-pen.ts

```diff
@@ -1,5 +1,4 @@
 import type {Empty, LinkedBundle} from '@devvit/protos'
-import {throttle} from '@devvit/shared-types/throttle.js'
```

Comment:
:no_bell: looks like shared-types is still needed for assets so we can't drop the dependency :+1: